graphics/opencv4/Portfile

```diff
@@ -5,9 +5,9 @@ PortGroup           cmake 1.1
 PortGroup           compiler_blacklist_versions 1.0
 PortGroup           github 1.0
 
-github.setup        opencv opencv 4.5.0
+github.setup        opencv opencv 4.5.1
```

Comment:
Steve, we're not ready to update to v4.5.1 quite yet, as that release has some unresolved build issues with the 'contrib' module.

Reply:
Thanks. Reverted.

Reply:
Are the python modules compatible with v4.5.0?